### PR TITLE
Change location tests

### PR DESCRIPTION
**Pull request by Safron1ta2**

### CODE REVIEW

**Review by Wolmeright — APPROVED** [review]

File: src/test/java/SafroN/Tests/ChangeLocationTestPositive.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        hotlineHeaderViewElements.locationBtn().click();
+
+        HotlineLocationViewHelper hotlineLocationViewHelper = new HotlineLocationViewHelper(driver);
+        hotlineLocationViewHelper.search("Київ");
+
+        HotlineLocationViewElements hotlineLocationViewElements = new HotlineLocationViewElements(driver);
+        hotlineLocationViewElements.locationKyiv().click();
+
+        HotlineHeaderViewHelper hotlineHeaderViewHelper = new HotlineHeaderViewHelper(driver);
+        hotlineHeaderViewHelper.getLocationText();
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Tests/ChangeLocationTestPositive.java:27]: 1. Rename test1() method, please
2. Group objects of the classes together at the top of the test1() method
3. Remove empty lines between steps (including openUrl() method)

File: src/test/java/SafroN/Tests/ChangeLocationTestPositive.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import BasePageAndTestInit.TestInit;
+import SafroN.Pages.HotlineHeaderViewElements;
+import SafroN.Pages.HotlineHeaderViewHelper;
+import SafroN.Pages.HotlineLocationViewElements;
+import SafroN.Pages.HotlineLocationViewHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ChangeLocationTestPositive extends TestInit {
+    //enter the correct location
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Tests/ChangeLocationTestPositive.java:12]: Remove comment

File: src/test/java/SafroN/Tests/ChangeLocationTestNegative.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        HotlineHeaderViewElements hotlineHeaderViewElements = new HotlineHeaderViewElements(driver);
+        hotlineHeaderViewElements.locationBtn().click();
+
+        HotlineLocationViewHelper hotlineLocationViewHelper = new HotlineLocationViewHelper(driver);
+        hotlineLocationViewHelper.search("SafroN");
+
+        HotlineLocationViewElements hotlineLocationViewElements = new HotlineLocationViewElements(driver);
+
+        Assert.assertTrue(hotlineLocationViewElements.exceptionMessage().isDisplayed());
+
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Tests/ChangeLocationTestNegative.java:24]: 1. Delete comment (line 11)
1. Rename test1() method, please
2. Group objects of the classes together at the top of the test1() method
3. Remove empty lines between steps (including openUrl() method)
4. Remove empty line (line 24)

File: src/test/java/SafroN/Pages/HotlineLocationViewElements.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HotlineLocationViewElements extends BasePage {
+    public HotlineLocationViewElements(WebDriver driver) {
+        super(driver);
+    }
+    private static final String SEARCH_LOCATION_FIELD = "//div[@class='m_b-lg']//input[@class='field']";
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Pages/HotlineLocationViewElements.java:11]: Add empty line before variables

File: src/test/java/SafroN/Pages/HotlineHeaderViewHelper.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package SafroN.Pages;
+
+import org.openqa.selenium.WebDriver;
+
+public class HotlineHeaderViewHelper extends HotlineHeaderViewElements{
+    public HotlineHeaderViewHelper(WebDriver driver) {
+        super(driver);
+    }
+    private String location;
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Pages/HotlineHeaderViewHelper.java:9]: Add empty line before variable

File: src/test/java/SafroN/Pages/HotlineHeaderViewElements.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        super(driver);
+    }
+
+    private static final String LOCATION_BTN = "//span[@class='city-name ellipsis']";
+    private static final String LOCATION_BTN_KYIV = "//span[contains(text(),'Київ')]";
+
+    public WebElement locationBtn(){
+        return $(LOCATION_BTN);
+    }
+    public WebElement locationBtnKyiv(){
```
Comment (AndriiMyronenko418) [inline src/test/java/SafroN/Pages/HotlineHeaderViewElements.java:18]: add empty line between the methods

**Review by AndriiMyronenko418 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**